requirements/system-requirements.md

```diff
@@ -29,19 +29,20 @@ This article lists the minimum hardware and software requirements for the instal
 
 The application can be installed on a workstation with __Windows 10, Windows 8 or Windows 7 operating system__, but you will not be able to load new SharePoint farm settings, but only connect to an existing SPDocKit database and open already saved farm settings.
```

Comment:
The application can be installed on a workstation with __Windows 10, Windows 8 or Windows 7 operating system__, but you will not be able to load new SharePoint farm settings, only connect to an existing SPDocKit database and open already saved farm settings. [one but too many]